### PR TITLE
Text fields new IA

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/text-fields.md [hunk 1]
```diff
@@ -132,17 +103,16 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/text-inputs/
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Required fields
+### Style
```
Comment (palindromebart) [inline _component_design/text-fields.md]: need to fix some redline issues in the visual. look for /style in the png folder.

File: _component_design/text-fields.md [hunk 2]
```diff
@@ -18,28 +18,13 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/text-inputs/
...
-{% endcolumn %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+## When to use
 
-{% figure [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/inputs-and-controls/text-fields/text-fields-hero-image.svg"/>
-{% endfigure %}
+Use text fields to enable users to enter text, numbers, passwords, emails, log in credentials, an API key, and any other data required by the user to complete a task.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/text-fields.md]: right, login - good catch.

**Comment by cyrichardson** [comment]
On it, @chri2547.

**Comment by chri2547** [comment]
Great! Thanks so much!

**Comment by cyrichardson** [comment]
@chri2547 Suggested changes for the text input file here: 
[Helix-Text-Inputs_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2488448/Helix-Text-Inputs_cyrEdits.docx)

Your link changes in the other 2 files LGTM. Let me know if you want for me to edit all of the content in those files as well (input selectors and progress bars). Thanks!

**Comment by palindromebart** [comment]
LGTM

File: _component_design/text-input.md [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Disabled
+
+In a disabled state, a text input cannot be edited by the user and the ``not-allowed`` style is applied to the cursor. The not-allowed cursor style prevents the user from clicking in the text input.
```
Comment (nimbinatus) [inline _component_design/text-input.md]:
```suggestion
In a disabled state, a text input cannot be edited by the user and the `not-allowed` style is applied to the cursor. The not-allowed cursor style prevents the user from clicking in the text input.
```

File: _component_design/progress-bar.md [hunk 4]
```diff
@@ -136,7 +136,7 @@ The minimal progress bar does not have state changes itself. When it appears, it
...
 
-- **Error states:** If an item has trouble loading, the progress bar itself does not convert to error state, but the pattern that the progress bar is a part of does. That pattern should follow our [text fields]({{site.baseurl}}/components/text-fields#states-overview) error state guidelines, which include presenting an error message with an error icon. From that error message, the user may choose to retry or eliminate the item processing.
+- **Error states:** If an item has trouble loading, the progress bar itself does not convert to error state, but the pattern that the progress bar is a part of does. That pattern should follow our [text input]({{site.baseurl}}/components/text-input.html) error state guidelines, which include presenting an error message with an error icon. From that error message, the user may choose to retry or eliminate the item processing.
```
Comment (nimbinatus) [inline _component_design/progress-bar.md]:
```suggestion
- **Error states:** If an item has trouble loading, the progress bar itself does not convert to error state, but the pattern that the progress bar is a part of does. That pattern should follow our [text input error state guidelines]({{site.baseurl}}/components/text-input.html#error), which include presenting an error message with an error icon. From that error message, the user may choose to retry or eliminate the item processing.
```

File: _component_design/text-input.md [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Prefixes and suffixes
+
+Use a prefix or suffix to clarify the meaning of the numeric input value entered by the user. Left justify a prefix and right justify a suffix.
```
Comment (nimbinatus) [inline _component_design/text-input.md]:
```suggestion
Use a prefix or suffix to clarify the meaning of the numeric input value entered by the user. Left align a prefix and right align a suffix.
```
See https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/left-justify-left-justified (which is one of the guides our style guide is based off of).

**Pull request merged**